.prettierignore

```diff
@@ -1,3 +1,12 @@
 artifacts
 cache
 node_modules
+types
+deployments
+temp-artifacts
+dist
+Certora
+*.json
+CHANGELOG.md
+.prettierignore
+.solcover.js
```

Comment:
should we included `.solcover.js`?
and `Certora`? At the end of the day, their source code should be the same the `contracts` one

Reply:
added to ignore to prevent more diffs but I understand your point